jobs/release.yaml

```diff
@@ -90,10 +90,12 @@
           values:
             - 0
             - 1
+            - 2
```

Comment:
this performs a snap upgrade from 
* 1.(xx-1)/stable to 1.xx/candidate
* 1.(xx-2)/stable to 1.xx/candidate